python/llm/src/ipex_llm/serving/fastapi/api_server.py

```diff
@@ -25,12 +25,20 @@
 import asyncio
 import uuid
 from typing import List, Optional, Union, Dict
+from .tgi_protocol import Parameters
 
 
 result_dict: Dict[str, str] = {}
 logger = logging.get_logger(__name__)
 
 
+class InputsRequest(BaseModel):
+    inputs: str
+    parameters: Optional[Parameters] = None
+    stream: Optional[bool] = False
+    req_type: str = 'completion'
+
+
 class PromptRequest(BaseModel):
```

Comment:
Maybe merge `PromptRequest` to `InputsRequest`?

Reply:
I will remove PromptRequest later